### DISCUSSION TITLE
Java 8 JRE

### DISCUSSION THREAD

**Discussion by laeubi** [eclipse-justj/justj#7] [Q&A]
Would it be possible to have a Java 8 JustJ (or even Java 9, 7, 6, 5 ...).

I know these are old, but for some setups it is quite good to have "real" JVMs ...

**Comment by merks** [comment]
I can only effectively release what's available here:

https://adoptium.net/temurin/releases/

The problem with Java 8 is that it doesn't support jlink so it would need to be a hand-crafted effort and only a full JRE would be possible without the modularization introduced after Java 8.

**Reply by laeubi** [reply]
They seem to have at least java 8, probably one can automate the P2 data by using `org.eclipse.equinox.p2.publisher.actions.JREAction` and then just publish the raw artifact? Modularization would then of course not be possible.

**Reply by merks** [reply]
Certainly a new separate mechanism would needed, and then the result would not be widely consumed given that EOL is coming up for Java 8 and that a Platform that can run on it is also years old and unsupported.  So the motivation and return-on-investment seem low...
